addons/crm/models/crm_team.py

```diff
@@ -608,13 +608,15 @@ def _action_update_to_pipeline(self, action):
             user_team_id = self.search([('id', '=', user_team_id)], limit=1).id
         else:
             user_team_id = self.search([], limit=1).id
-            action['help'] = _("""<p class='o_view_nocontent_smiling_face'>Add new opportunities</p><p>
-    Looks like you are not a member of a Sales Team. You should add yourself
-    as a member of one of the Sales Team.
-</p>""")
+            action['help'] = _("<p class='o_view_nocontent_smiling_face'>Create an Opportunity</p>")
```

Comment:
While modifying this code, I think we should take the opportunity to extract sentences to translate independently from the html itself.

Reply:
Indeed we can do that for a single sentence. But in the changes below the condition, we need to inject parts of the sentence between html tags (bold, anchor...). And if we translate parts separately, aren't they more likely to be broken grammatically?

Reply:
We could keep the bold tags (not that complicated) and already remove extra <p> for example.

Reply:
In short: ok, we can just add the p tags as extra, the remaining is probably better kept as it is.